src/scss/constants.scss

```diff
@@ -155,6 +148,20 @@ $image-input-upload-button-background: #462a87;
 $image-input-upload-button-background--focus: rgba(55, 90, 127, 0.18);
 $image-input-border--error: 2px solid red;
 
+// NOTE: Magic number determined empirically to fill the footer space in default case
+$admin-ng-sidebar-footer-buttons-width--magic-number: 11.5vw;
+$admin-ng-sidebar-width--normal: 30vw;
+$admin-ng-sidebar-width--min: $admin-ng-sidebar-footer-buttons-width--magic-number;
+$admin-ng-sidebar-width--max: 30vw;
+$admin-ng-sidebar-footer-height--max: 4rem;
+$admin-ng-showcase-width--normal: calc(
+  100vw - #{$admin-ng-sidebar-width--normal}
+);
+$admin-ng-tab-bar-height: calc(2rem + #{$spacing--sm});
+$admin-ng-panel-height: calc(
+  100vh - #{$admin-ng-tab-bar-height} - #{$reserved-page-height}
+);
+
```

Comment:
So our AdminSidebar depends on a lot of vars. Is there any option to decrease this number?
It's fine if 8 is minimum number of good styling

Reply:
Not at the moment. 

[OOS]
Best we can do is start splitting `constants.scss` into directory with `constants/index.scss` and importing into the index from other files in it, one might be named `constants/admin.scss`

Reply:
The issue with unifying existing codebase is the extraction of common patterns before making one general Admin component and just plug in `{children}` for the side bar and main page. Still not there.